sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -426,6 +426,9 @@ export function setClientRequestIdPolicy(requestIdHeaderName?: string): Pipeline
 // @public
 export const setClientRequestIdPolicyName = "setClientRequestIdPolicy";
 
+// @public
+export type StreamBuilder = () => ReadableStream<Uint8Array> | NodeJS.ReadableStream;
```

Comment:
Nit: "Builder" as a suffix feels like it should be some kind of chainable primitive to manufacture a type rather than a specific type alias for this scenario.

Would it be fair to call it a "ResettableStream" instead?

Reply:
Good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/27743/commits/a3aaa3018129a1b42f8bf79154822b08a8498372 as discussed offline.